mnefun/_mnefun.py

```diff
@@ -610,18 +610,24 @@ def fetch_raw_files(p, subjects, run_indices):
         raw_dir = op.join(subj_dir, p.raw_dir)
         if not op.isdir(raw_dir):
             os.mkdir(raw_dir)
-        finder_stem = 'find %s ' % p.acq_dir
-        # build remote raw file finder
-        fnames = get_raw_fnames(p, subj, 'raw', True, False, run_indices[si])
+        fnames = get_raw_fnames(p, subj, 'raw', True, False,
+                                run_indices[si])
         assert len(fnames) > 0
-        finder = (finder_stem +
+        # build remote raw file finder
+        for remote_dir in ['/sinuhe_data01', '/data101', '/sinuhe/data01',
+                           '/sinuhe/data02','/sinuhe/data03']:
```

Comment:
Where are you putting that conditional? Maybe it would help if you pushed code with updated example that you think should work, but errors out